backend/src/controllers/reports/helpers/index.js

```diff
@@ -57,36 +71,59 @@ const getDocumentApiBody = async (
   };
 };
 
+// helper functions for getDocumentApi()
+const isValidInput = ({ templateType, templateFileName }) =>
+  templateFiletypes.includes(templateType) &&
+  templateFiletypes.includes(templateFileName.slice(-4));
+
+const getTemplatePath = ({ templateType, templateFileName }) =>
+  `../../../../reports/${templateType}/${templateFileName}`;
+
+// decide whether to export an XLSX or PDF
+const getOutputFormat = (templateType) =>
```

Comment:
This will work, though there is issue with this, in that if there is no match, there will be no default, i'm assuming it would pass an undefined.   the one place i feel like tests is required is in helper functions.  What if we give them the opportunity to upload a template, and instead of an xlsx file the uploaded and xls file, it should still work, but by this logic it would fail.  We might want to just indicate a todo, as this function will mature and change

Reply:
since we don't enable template uploads from the user, I'll make a comment on the function for future devs so they know where to look when (if) this feature is added.